extensions/amp-subscriptions/amp-subscriptions.md

```diff
@@ -262,97 +483,215 @@ The vendor service configuration must reference the service ID and can contain a
+
+### Fallback entitlement
+
+If all configured services fail to get the entitlements, the entitlement configured under `fallbackEntitlement` section will be used as a fallback entitlement for `local` service. The document is unblocked based on this fallback entitlement.
+
+Example fallback entitlement:
 ```js
 {
-  "granted": true/false,
-  "grantReason": "SUBSCRIBER/METERING",
-  "data" : {...}
+  "fallbackEntitlement": {
+    "source": "fallback",
+    "granted": true,
+    "grantReason": "SUBSCRIBER",
+    "data": {
+      "isLoggedIn": false
+    }
+  }
 }
 ```
 
-The properties in the Entitlement response are:
+### Structured data markup
 
-- `granted` - boolean stating if the access to the document is granted or not.
+`amp-subscriptions` relies on the Schema.org page-level configuration for two main properties:
 
-- `grantReason` - the string of the reason for giving the access to the document, recognized reasons are either SUBSCRIBER meaning the user is fully subscribed or METERING meaning user is on metering.
+1.  The product ID that the user must be granted to view the content.
+2.  Whether this content requires this product at this time.
 
-- `data` - free-form data which can be used for template rendering, e.g. messaging related to metering or article count.
+The JSON-LD and Microdata formats are supported.
 
-In cases where the access is granted by a means other than the Entitlement response, messaging via the data property may not be seen by the user. Do not use `data` for granting/denying access to content, conditional display of content based on user access, or displaying user or account related information.
+More detail on the markup is available [here](https://developers.google.com/search/docs/data-types/paywalled-content).
 
-Notice, while it's not explicitly visible, all vendor services also implement authorization endpoints of their own and conform to the same response format.
+#### JSON-LD markup
 
-## Pingback endpoint
+Using JSON-LD, the markup would look like:
 
-Pingback is an endpoint provided by in the "local" service configuration and called by the AMP Runtime. It is a credentialed CORS POST endpoint. AMP Runtime calls this endpoint automatically when the Reader has started viewing the document. One of the main goals of the Pingback is for the Publisher to update metering information.
+```html
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "NewsArticle",
+    "isAccessibleForFree": false,
+    "publisher": {
+      "@type": "Organization",
+      "name": "The Norcal Tribune"
+    },
+    "hasPart": {...},
+    "isPartOf": {
+      "@type": ["CreativeWork", "Product"],
+      "name" : "The Norcal Tribune",
+      "productID": "norcal_tribune.com:basic"
+    }
+  }
+</script>
+```
 
-Pingback is optional. It's only enabled when the "pingbackUrl" property is specified.
+Thus, notice that:
```

Comment:
Why am I noticing this? Why is it important?

Reply:
I've merged the two examples as there was repetition.